src/components/HeroCheckbox/HeroCheckbox.css

```diff
@@ -27,6 +27,10 @@
   color: #cb8468;
 }
 
+.text-cassidy {
+  color: #d44124;
```

Comment:
Man, we need a way to see hex colors in a PR diff! 
![image](https://user-images.githubusercontent.com/82317/201138674-51fab26f-d41c-4cd9-b4a3-545a93f661ea.png)